src/extension.ts

```diff
@@ -9,9 +9,11 @@ import {
 
 import {
     LanguageClient, LanguageClientOptions, SettingMonitor, ServerOptions, TransportKind, TextDocumentIdentifier, Location as LSLocation,
-    Position as LSPosition, RevealOutputChannelOn, DynamicFeature
+    Position as LSPosition, RevealOutputChannelOn, DynamicFeature, BaseLanguageClient
```

Comment:
why do we need to import `BaseLanguageClient`? can't the `LanguageClient` be used?